app/src/main/java/nerd/tuxmobil/fahrplan/congress/utils/FahrplanMisc.kt

```diff
@@ -0,0 +1,71 @@
+package nerd.tuxmobil.fahrplan.congress.utils
+
+import android.app.AlarmManager
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import nerd.tuxmobil.fahrplan.congress.MyApp
+import nerd.tuxmobil.fahrplan.congress.alarms.AlarmReceiver
+import nerd.tuxmobil.fahrplan.congress.alarms.AlarmUpdater
+import nerd.tuxmobil.fahrplan.congress.extensions.*
+import nerd.tuxmobil.fahrplan.congress.models.DateInfo
+import nerd.tuxmobil.fahrplan.congress.models.DateInfos
+
+
+object FahrplanMisc {
+
+    private const val LOG_TAG = "FahrplanMisc"
+
+    /**
+     * Returns a [DateInfos] instance composed from the given `dateInfos` list.
```

Comment:
In KDoc you can reference parameters with square brackets. Please use them for `[dateInfos]` instead of the backticks.